### ISSUE TITLE
Malformed metrics

### ISSUE THREAD

**Issue by arieradle** [PrometheusClientNet/Prometheus.Client#68]
Hello,

I observe the following malformed metrics output: 
```
some_metic1{lab1="f5378719-907e-11e9-b826-0050569bce07"} 29520800
some_metic1_count{lab1="f5378719-907e-11e9-b826-0050569bce07"} 103lab2="Chat",lab1="f5378719-907e-11e9-b826-0050569bce07",quantile="0.9"} NaN
```

It seems that part of the line of a metric is somehow missing from the output.

**Comment by phnx47** [comment]
Fixed in 3.0.1: https://github.com/PrometheusClientNet/Prometheus.Client/issues/66

@arieradle You have problem in 3.0.1?

**Comment by arieradle** [comment]
Thanks, my bad
